nixos/modules/system/boot/systemd.nix

```diff
@@ -725,6 +725,8 @@ in
 
       "systemd/system.conf".text = ''
         [Manager]
+        DefaultCPUAccounting=yes
+        DefaultBlockIOAccounting=yes
```

Comment:
When testing on my own system, the CPU accounting didn't seem to work without this setting. (But worked just fine even though I accidentally put it in user.conf instead of system.conf the first time, so I'm not sure what's going on.)

I can't test the IO accounting, you may well be right about that. What should I be using instead?